tools/SetupFlow/DevHome.SetupFlow/Views/PackageCatalogListView.xaml

```diff
@@ -1,43 +1,43 @@
     xmlns:views="using:DevHome.SetupFlow.Views"
-    mc:Ignorable="d">
-    <UserControl.Resources>
-        <ResourceDictionary>
-            <!-- Number of items to display per panel -->
-            <x:Int32 x:Key="ItemsPerPanel">6</x:Int32>
-        </ResourceDictionary>
-    </UserControl.Resources>
-    <StackPanel>
-        <!-- Package catalogs -->
-        <ItemsRepeater
-            ElementPrepared="ItemsRepeater_ElementPrepared"
-            ItemsSource="{x:Bind ViewModel.PackageCatalogs, Mode=OneWay}">
-            <ItemsRepeater.ItemTemplate>
-                <DataTemplate x:DataType="viewmodels:PackageCatalogViewModel">
-                    <!-- Border configuration is applied in code-behind -->
-                    <Border>
-                        <views:PackageCatalogView Catalog="{x:Bind Mode=OneWay}" GroupSize="{StaticResource ItemsPerPanel}"/>
-                    </Border>
-                </DataTemplate>
-            </ItemsRepeater.ItemTemplate>
-            <ItemsRepeater.Layout>
-                <StackLayout Spacing="10" />
-            </ItemsRepeater.Layout>
-        </ItemsRepeater>
-
-        <!-- Package catalog shimmers -->
-        <ItemsRepeater ItemsSource="{x:Bind ViewModel.PackageCatalogShimmers, Mode=OneWay}">
-            <ItemsRepeater.ItemTemplate>
-                <DataTemplate>
-                    <views:ShimmerPackageCatalogView PackageCount="{StaticResource ItemsPerPanel}" />
-                </DataTemplate>
-            </ItemsRepeater.ItemTemplate>
-        </ItemsRepeater>
-    </StackPanel>
-</UserControl>
+    mc:Ignorable="d">
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <!-- Number of items to display per panel -->
+            <x:Int32 x:Key="ItemsPerPanel">6</x:Int32>
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <StackPanel>
+        <!-- Package catalogs -->
+        <ItemsRepeater
+            ElementPrepared="ItemsRepeater_ElementPrepared"
+            ItemsSource="{x:Bind ViewModel.PackageCatalogs, Mode=OneWay}">
+            <ItemsRepeater.ItemTemplate>
+                <DataTemplate x:DataType="viewmodels:PackageCatalogViewModel">
+                    <!-- Border configuration is applied in code-behind -->
+                    <Border>
+                        <views:PackageCatalogView Catalog="{x:Bind Mode=OneWay}" GroupSize="{StaticResource ItemsPerPanel}"/>
+                    </Border>
+                </DataTemplate>
+            </ItemsRepeater.ItemTemplate>
+            <ItemsRepeater.Layout>
+                <StackLayout Spacing="10" />
+            </ItemsRepeater.Layout>
+        </ItemsRepeater>
+
+        <!-- Package catalog shimmers -->
+        <ItemsRepeater ItemsSource="{x:Bind ViewModel.PackageCatalogShimmers, Mode=OneWay}">
+            <ItemsRepeater.ItemTemplate>
+                <DataTemplate>
+                    <views:ShimmerPackageCatalogView Margin="0,0,0,18" PackageCount="{StaticResource ItemsPerPanel}" />

```

Comment:
Any reason for the 18?

Reply:
The margin value here attempts to make the shimmer animation a little nicer since today consecutive shimmers are rendered without gap in between.

<img src="https://github.com/microsoft/devhome/assets/104940545/df8afcd1-5663-4d17-8842-b2663709c744" Width="100" />